### ISSUE TITLE
cancelTag(Object) throws NoSuchElementException

### ISSUE THREAD

**Issue by longkai** [square/picasso#919]
I rarely encounter this problem, when Picasso call cancalTag(Object), in this line ``List<Action> actions = new ArrayList<Action>(targetToAction.values());`` the targetToAction(a WeahHashMap)' s values() method throws it.

I searched for a while, it may be the the [apache harmony' s bug](https://issues.apache.org/jira/browse/HARMONY-3883?attachmentOrder=desc), but in Picasso lib, how can we avoid it?

**Comment by JakeWharton** [comment]
Yep. Had another report of this just this week and I came to the same conclusion!

**Comment by ruslanle** [comment]
Are there any ways to avoid it?
